src/constants.ts

```diff
@@ -1,18 +1,7 @@
 import { RepoType } from "./types/Basic";
 export const TIME_LABELS: string[] = ["Time: <1 Hour", "Time: <1 Day", "Time: <1 Week", "Time: <2 Weeks", "Time: <1 Month"];
-
-export const repoMapping: RepoType[] = [
-  {
-    group: -1001738021587,
-    github: "Seprintour-Test/test",
-  },
-  {
-    group: -1001558587400,
-    github: "ubiquity/testing-telegram-ubiquibot",
-  },
-];
+export const GITHUB_PATHNAME = "/github-link";
```

Comment:
What is `/github-link` ?

Reply:
It is the command for users to bind their telegram with github

Reply:
Short commands are more attractive. Perhaps we can do `/register` ?

Reply:
> Short commands are more attractive. Perhaps we can do `/register` ?

I'd use this